arch/arm/boot/dts/Hawaii.dts

```diff
@@ -0,0 +1,210 @@
+/dts-v1/;
```

Comment:
Can we stay clean, again, and remove anything that is not MACH_JAVA and Sony stuff?
Maybe it'd be okay if we don't support other old Broadcom CPUs :)